Product plan/Product-plan.tex

```diff
@@ -83,62 +85,93 @@ \subsubsection*{Could haves}
+
+
+\subsubsection*{Week 7}
+This week we will mainly focus on implemented an easier way to debug with emotions. This includes creating a debug tool so that programmers can check what is going wrong and what is happening with the emotions at a given moment in time. We will try to extend the Simple IDE \cite{SimpleIDE} so that it includes the emotional database.
+After we released the first version we  aim for the ease of use of the implemented specifications. Therefore we will create the debug tool for the other group.(3)
+
+\subsubsection*{Week 8}
+This week we will focus on updating relations on the fly. This can be handy for programmers. Also we have time left to fix tasks that are not finished but are very important. (7)
+
+\subsubsection*{Week 9}
+This week we will also fix, test and finish incomplete tasks. Also this week we will finish the extra documentation on how to use the integration. It will be worked on continuously but this week these documents will be completed.(8,9)
+
+\subsubsection*{Week 10}
+During this week the presentations will be made and all things related to it will be completed.
 
 \section{Product backlog}
 In this section you can find an overview of the backlog and release plan.
 \subsection{User}
 We define a user as a programmer that wants to use GOAL to develop an AI identity. While doing this, he wants to involve emotions and develop his identity in such a way that he can use emotions to trigger actions, but also actions in the environment (via percepts) that change his emotions would be very handy for the user. The user has a good understanding of the programming language GOAL. He has a very small understanding of Gamygdala.
 \subsection{User stories of features}
-\begin{itemize}
-\item As a programmer I want to be able to define a relationship between beliefs and goals so that these effect the emotional state of the agent. It must be similar to define goals in GOAL.
+\subsubsection*{1}
+As a programmer I define a relation between a belief and a goal in the emotion configuration. When I run the agent, each cycle the emotions is updated according to the defined relations. 
 
-\item As a programmer I want to be able to define relations between different agents.
+\subsubsection*{2}
+As a programmer if I define a relation between to agents in the configuration, when I run this configuration, Gamygdala is aware of this relation and updates the emotion of both agents accordingly.
+
+\subsubsection*{3}As a programmer when I run an emotional agent with the Simple IDE\cite{SimpleIDE} I can to inspect its emotional database.
```

Comment:
"when I run an emotional agent with the Simple IDE I'm able to inspect its emotional database"